python/vowpalwabbit/sklearn_vw.py

```diff
@@ -742,14 +980,24 @@ class would be predicted.
         >>> X = np.array([ [10, 10], [8, 10], [-5, 5.5], [-5.4, 5.5], [-20, -20],  [-15, -20] ])
         >>> y = np.array([1, 1, 2, 2, 3, 3])
         >>> from vowpalwabbit.sklearn_vw import VWMultiClassifier
-        >>> model = VWMultiClassifier(oaa=3, loss_function='logistic')
+        >>> model = VWMultiClassifier(oaa=2, loss_function='logistic')
```

Comment:
what's the reason of changing this value?

Reply:
ah good catch, that should be reverted to 3 (or the y values should be updated to only 2). I was changing tests to use 2, because there was a bug with sklearn tests that had > 2 labels and this probably got caught in that change.

Reply:
Looks like I jumped the gun with merging! I can submit a PR to fix this if you'd like?

Reply:
sure, it's just a docstring, but would be good to put in a pr quickly to flip this to 3

Reply:
No prob, worth fixing also so we don't forget later. PR here: https://github.com/VowpalWabbit/vowpal_wabbit/pull/2482